backend/src/pilcom_cli/mod.rs

```diff
@@ -0,0 +1,93 @@
+mod json_exporter;
+
+use crate::{Backend, BackendBuilder, Proof};
+use ast::analyzed::Analyzed;
+use json::JsonValue;
+use number::{write_polys_file, DegreeType, FieldElement};
+use std::{
+    fs,
+    io::{self, BufWriter},
+    path::Path,
+};
+
+pub struct PilcomCliBuilder;
+
+impl<T: FieldElement> BackendBuilder<T> for PilcomCliBuilder {
+    fn setup_from_params(&self, size: DegreeType) -> Box<dyn Backend<T>> {
+        Box::new(PilcomCli { degree: size })
+    }
```

Comment:
Same here, estarks don't need setup, so this shouldn't be possible by construction.